views/layer/vector.html

```diff
@@ -12,7 +12,7 @@
         </div>
         <div class="d-flex col">
           <select class="custom-select" ng-model="vector.preset"
-              ng-options="preset.label for preset in vector.presets track by preset.id">
+              ng-options="(preset.label + (preset.default === true ? ' (default)' : '') + (preset.published === true ? '' : ' (unpublished)')) for preset in vector.presets track by preset.id">
```

Comment:
This text seems like it could get a bit long. For the "Unpublished" state, I feel like displaying an asterisk and maybe altering the tooltip could maybe work? Similar to how a text editor indicates on its tab that your file has unsaved changes.

Reply:
I'll switch over to the asterisk for now. If we want to do an <option ng-repeat="..."> instead for more flexibility than <select ng-options="..."> gives us (e.g. tooltips, custom classes, etc) then we can cross that bridge later.